src/contracts/interfaces/IEigenPod.sol

```diff
@@ -79,29 +83,37 @@ interface IEigenPod {
     /// @notice an indicator of whether or not the podOwner has ever "fully restaked" by successfully calling `verifyCorrectWithdrawalCredentials`.
     function hasRestaked() external view returns (bool);
 
-    /// @notice block number of the most recent withdrawal
-    function mostRecentWithdrawalBlockNumber() external view returns (uint64);
+    /// @notice block timestamp of the most recent withdrawal
+    function mostRecentWithdrawalTimestamp() external view returns (uint64);
+
+    /// @notice Returns the validatorInfo struct for the provided pubkeyHash
+    function validatorPubkeyHashToInfo(bytes32 validatorPubkeyHash) external view returns (ValidatorInfo memory);
 
 
     ///@notice mapping that tracks proven partial withdrawals
-    function provenPartialWithdrawal(uint40 validatorIndex, uint64 slot) external view returns (bool);
+    function provenWithdrawal(bytes32 validatorPubkeyHash, uint64 slot) external view returns (bool);
+
+    /// @notice this is a mapping of validator indices to a Validator struct containing pertinent info about the validator
+    function validatorStatus(bytes32 pubkeyHash) external view returns(VALIDATOR_STATUS);
```

Comment:
documentation seems wrong? this is pulling a specific field out of the struct, right?

Reply:
fixed